src/main/java/com/coumin/woowahancoupons/coupon/dto/CouponCreateRequestDto.java

```diff
@@ -9,32 +16,48 @@
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class CouponCreateRequestDto {
 
+    @Size(min = 1, max = 100)
+    @NotNull
     private String name;
 
+    @Max(10_000)
     private Long amount;
 
+    @Max(100_000)
+    @PositiveOrZero
     private Long minOrderPrice;
 
+    @NotEmpty
```

Comment:
```suggestion
    @NotBlank
```
NotEmpty는 " "은 허용이라 NotBlank 사용을 추천드립니다!

Reply:
완료!